converter/internal/promtailconvert/testdata/source_pipeline_example.river

```diff
@@ -0,0 +1,41 @@
+loki.write "default_0" {
+	endpoint {
+		url              = "http://localhost/loki/api/v1/push"
+		follow_redirects = false
+		enable_http2     = false
+	}
+	external_labels = {}
+}
```

Comment:
tiny nit: it would be nice to put loki.write at the bottom. I typically see the order of the files as:

1. `discovery.*` component(s)
2. `discovery.relabel` component(s)
3. `loki.source.*` component(s) 
4. `loki.write` component(s)